src/post_process/m_global_parameters.fpp

```diff
@@ -276,6 +276,13 @@ contains
         bc_y%beg = dflt_int; bc_y%end = dflt_int
         bc_z%beg = dflt_int; bc_z%end = dflt_int
 
+        #:for DIM in ['x', 'y', 'z']
+            #:for DIR in [1, 2, 3]
+                bc_${DIM}$%vb${DIR}$ = 0d0
```

Comment:
should this not be `dflt_real`?

Reply:
The way these BCs are implemented requires the default to be zero. See here: https://github.com/wilfonba/MFC-Wilfong/blob/4cdb21bece7cb10f3be3c785302bf668781147f6/src/simulation/m_boundary_conditions.fpp#L1051-L1072